pkg/visor/init.go

```diff
@@ -344,6 +345,59 @@ func initDmsgCtrl(ctx context.Context, v *Visor, _ *logging.Logger) error {
 	return nil
 }
 
+func initDmsgHTTPLogServer(ctx context.Context, v *Visor, log *logging.Logger) error {
+	dmsgC := v.dmsgC
+	if dmsgC == nil {
+		return nil
+	}
+	var dmsgTimeout = v.conf.LogRotationInterval
```

Comment:
The log rotation interval should not be used as the dmsg write timeout. Log rotation interval determines what files will be cleaned up from generated log files. So if rotation interval is 2 days, all files older than 2 days will be cleaned up.

Reply:
Will change